### DISCUSSION TITLE
Outgoing SMTP

### DISCUSSION THREAD

**Discussion by Acenl12** [docker-mailserver/docker-mailserver#3893] [Q&A]
I have enabled SSL certificates using manual mode, but when sending outgoing mail TLS is not being used and everything goes over port 25.
How can I configure outgoing smtp tls?

**Comment by casperklein** [comment]
If the destination mail server supports encryption, the transport is encrypted using STARTTLS.

**Comment by polarathene** [comment]
> sending outgoing mail TLS is not being used

If you are sending it to a well established service like Gmail TLS should be used.

It has nothing to do with your own certificate (only relevant for mail being sent to you), just that the other mail server you connect to agrees to use TLS via STARTTLS, and that you both have a compatible cipher to use for that. DMS requires TLS 1.2 minimum.

---

> everything goes over port 25

That's expected. If you are relaying the default port is 25, if you need to change that then configure it as per our docs.
